### ISSUE TITLE
Locked together

### ISSUE THREAD

**Issue by kschoenberger08** [Bumblefuck/25mm-olfa-blade-holder-knife#3]
All pieces are locked together, since it is one STL file. The orientation of them is odd and I can't flip them all so they are all flat on the build plate...

<img width="329" alt="IMG_9555" src="https://github.com/user-attachments/assets/85bf0e54-412c-4d75-8a69-6c6bb556d1c1">

<img width="470" alt="image" src="https://github.com/user-attachments/assets/3c61aa4b-a26a-4245-8284-50cd2363555a">

**Comment by Pyr-000** [comment]
If you happen to be using PrusaSlicer or any of the other Slic3r-based slicers you can right-click an object and separate the bodies.
This looks like Cura, though, which apparently needs a plugin to be able to do this.

Zip of the split bodies re-exported: [25 mm olfa blade holder knife split.zip](https://github.com/user-attachments/files/16564727/25.mm.olfa.blade.holder.knife.split.zip)

**Comment by 62165** [comment]
> If you happen to be using PrusaSlicer or any of the other Slic3r-based slicers you can right-click an object and separate the bodies. This looks like Cura, though, which apparently needs a plugin to be able to do this.
> 
> Zip of the split bodies re-exported: [25 mm olfa blade holder knife split.zip](https://github.com/user-attachments/files/16564727/25.mm.olfa.blade.holder.knife.split.zip)

Nice. I didn't know you could do this. I wasn't able to open the zip, but was able to download a plugin to split them!

**Comment by h-dub** [comment]
the angled build is solving an important safety issue:
if it is printed flat, the blade can put force at the weak spot between two layer lines. Could result in catastrophic failure if the print delaminated while someone was pressing hard. with the angled print this is less of an issue as multiple layers would have to shear for a catastrophic failure.

I printed it flipped 180 from the way you show with tree supports

**Comment by 1organoleptic** [comment]
> I printed it flipped 180 from the way you show with tree supports

Tree supports definitely look like the way to go for this.

**Comment by Bumblefuck** [comment]
Thanks! I'd post the fusion file for re-spin but there's too much secret stuff in the file header.
OBJ files are over 25MB so Github won't let me post. It's a whole fuck around.

**Comment by Bumblefuck** [comment]
This is the orientation I print in to mitigate the hassle of support stuck in the slot.

**Comment by Leopere** [comment]
> Thanks! I'd post the fusion file for re-spin but there's too much secret stuff in the file header.
> OBJ files are over 25MB so Github won't let me post. It's a whole fuck around.

Dunno  if you can enable LFS (large file support) might help but like if its easier at some point I can submit a PR to split up the components I just feel like the channel for the actual blad isn't quite wide enough and I keep snapping my little testies bits that lock the slide in place when trying to move it around I tried printing PLA+ black from Elegoo at .15 layer hight on a .4 nozzle via the prusa mk3s+

**Comment by sypernova6969** [comment]
@Bumblefuck 
have you tried to print it with the "bottom"  on the bed?  I.E. standing up? I mean that narrow part where you insert the blade into? I know it`s not ideal, but it would make is so that no supports are needed..  
Also.  are you ok???  that last video was.  Cryptic as fuck.  Let us know if we can help... in case help is needed.

**Comment by Leopere** [comment]
I actually was able to get it to print nicely on the flat end with organic supports but the channel for the blade was a touch too tight to let it freely slide so now I just kind of have a plastic thing with a blade jammed in it.  3d printing is awesome but man it can kinda stink de-kink-ifying things.

**Comment by Leopere** [comment]
<img width="1629" alt="image" src="https://github.com/user-attachments/assets/e94be773-e870-44b6-9290-cc570fa85197"> I'm just gonna leave this here.

**Comment by rjorgenson** [comment]
It's at least possible to print in the same orientation as Leopere with no supports and no brim. Gotta have good adhesion and i'm not sure it would work on a bed slinger but I was able print this on an x1c with no issues. 0.6mm Nozzle, 0.18mm layer height, PLA-CF, all default settings other than forcing no brim.

![image](https://github.com/user-attachments/assets/a3c3e41c-98a2-40e0-87f4-33c6fe5139e1)

![image](https://github.com/user-attachments/assets/73ca72e0-6d2c-4777-9303-0ea77f7a54c2)

**Comment by Leopere** [comment]
@rjorgenson do you find the blade slides okay? I feel like I get a bit of elephant foot on the peggy thing that's supposed to slide on that back there which seems to cause some stickage.

**Comment by rjorgenson** [comment]
@Leopere Yeah I haven't had any issues with fitment of the blade, there's no detectable elephants foot on my print. The only fitment issue i had was the backing pin behind the blade rubbing against the back of the body, seems to be because of the textured plate I printed on it's much smoother after messing around with it for an evening. The tolerances do seem very tight on this, my guess is the model designed for CNC wasn't changed much for the 3d print version. You could try setting your flow to like .98 or something and see if that solves the elephants foot and the clearance in the blade channel. The other thing that could be causing fitment issues of the blade is the blade thickness itself. I have olfa 406605 blades (https://olfa.com/products/25mm_hbb_black_snap_blades?variant=40584946057399) other blades might be slightly thicker causing friction even if your flow is accurate to the model.

![IMG_7970](https://github.com/user-attachments/assets/887379d9-d565-4e06-aeb4-bc21b459b99d)

**Comment by darrendignam** [comment]
https://drive.google.com/file/d/1UQBe-5EL0sFED54YIUa5jSlx4GH-HlBG/view?usp=sharing

File for bambu labs. Slice: 6 walls, 100% infill, no supports. Blade groove is clear. PArts need a bit of finishing to get assembled.
